tools/android_custom_build/scripts/build.sh

```diff
@@ -6,21 +6,29 @@
 
 set -e -x
 
-USAGE_TEXT="Usage: ${0} <build config> <ops config file> <build settings file> <output directory>"
+USAGE_TEXT="Usage: ${0} <build config> <output directory> <build settings file> [<ops config file>]"
 
 BUILD_CONFIG=${1:?${USAGE_TEXT}}
-OPS_CONFIG_FILE=${2:?${USAGE_TEXT}}
+OUTPUT_DIR=${2:?${USAGE_TEXT}}
 BUILD_SETTINGS_FILE=${3:?${USAGE_TEXT}}
-OUTPUT_DIR=${4:?${USAGE_TEXT}}
+OPS_CONFIG_FILE=${4}  # optional input
 
+# build in directory that is not shared with the host to avoid permissions issues and speed up file access
 BUILD_DIR=/workspace/build
 
 # build ORT AAR
-python3 /workspace/onnxruntime/tools/ci_build/github/android/build_aar_package.py \
-  --build_dir="${BUILD_DIR}" \
-  --config="${BUILD_CONFIG}" \
-  --include_ops_by_config="${OPS_CONFIG_FILE}" \
-  "${BUILD_SETTINGS_FILE}"
+if [[ -n "${OPS_CONFIG_FILE}" ]]; then
+  python3 /workspace/onnxruntime/tools/ci_build/github/android/build_aar_package.py \
+    --build_dir="${BUILD_DIR}" \
+    --config="${BUILD_CONFIG}" \
+    --include_ops_by_config="${OPS_CONFIG_FILE}" \
```

Comment:
Should we add `--enable_reduced_operator_type_support` by default? It's slightly harder for the user to discover this option without it existing in the mobile build config, and it shouldn't hurt.

Reply:
sure, good idea

Reply:
hm, it wasn't as simple as I thought. build_aar_package.py doesn't currently pass `--enable_reduced_operator_type_support` through. we could add that, but it wouldn't work for older versions of ORT.

I added some help text to the android custom build script to mention use of the `--enable_reduced_operator_type_support` option when providing a reduced ops config file.